README.md

```diff
@@ -6,7 +6,7 @@ Quickmail is a Moodle block that provides selective, bulk emailing within course
 
 ## Requirements
 
-- Moodle 3.3 (build 2017051500 or later)
+- Moodle 3.4 (build 2017111300or later)
```

Comment:
shouldn't this be 3.5?

Reply:
Yes, and I accidentally removed the space between the version string and "or". I think I was originally planning to drop 3.3 only.